examples/seismic/acoustic/acoustic_example.py

```diff
@@ -95,6 +107,8 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
     parser.add_argument("-dle", default="advanced",
                         choices=["noop", "advanced", "speculative"],
                         help="Devito loop engine (DSE) mode")
+    parser.add_argument("--c", default=False, dest='constant', action='store_true',
```

Comment:
Can we make the argument signature `'-c', '--constant'` for consistency? You won't need to specify `dest='constant'` then.

Reply:
sure